packages/client/src/App.sass

```
@@ -21,6 +21,7 @@ $white-20: rgba(255,255,255, 0.2)
 $grey-primary: #939393
 $grey-dark: #6D6D6D
 $grey-secondary: #DDDDDD
+$grey-light: #DDDDDD // Override bulma default to match designs
```

```suggestion
$grey-light: #DDDDDD
```
we expect some of these to override bulma defaults (such as `grey-dark`) so comment is redundant